src/commands/delete.ts

```diff
@@ -63,7 +62,8 @@ async function doDelete(items: TektonNode[], toRefresh: Refreshable, commandId?:
         toDelete.set(item, deleteCommand);
       }
       if (refResource && checkRefResource()) {
-        if (referenceOfTaskAndClusterTaskInCluster(item, taskRunList)) {
+        const pipelineList = await getPipelineList();
```

Comment:
> Other point is that in the `if` the `refResource` can be omitted because it is always true (there is only the initialization before this point), so you can only check the setting `checkRefResource`.

@lstocchi I am using `refResource` for multiple resource delete for example if I have selected 4 task to delete and all the task is being used in pipleine, if I find one task which is used in pipeline then I will skip the detection process for other resources.